src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="card-image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+      <span class="product-card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </span>
+      <span class="product-card__code">Код товара: 195434</span>
+      <div class="product-card__rating">
+        <div class="product-card__stars">
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+        </div>
+        <span class="product-card__feedback">Отзывов: 5</span>
```

Comment:
Здесь должна быть **ссылка**, поскольку на реальном сайте - это переход на страницу с отзывами.
```suggestion
        <a href="#" class="product-card__feedback">
          Отзывов: 5
        </span>
```